libraries/chain/include/sysio/chain/authority.hpp

```diff
@@ -26,7 +26,7 @@ struct shared_public_key {
             public_key_storage = pub;
          }
       }, pubkey);
-      return std::move(public_key_storage);
+      return public_key_type(std::move(public_key_storage));
```

Comment:
Should rename `k1r1em` to `shim` or `k1r1emedbls`. I think `shim` is the better option.